### PR TITLE
MNT-22154 : [Security] Share proxy allows unauthenticated access to repository webscripts

### PR DESCRIPTION
**Pull request by alexbalmus**
- Additional properties for endpointController bean in slingshot-application-context.xml
   - Update Webscripts dependency: 8.12 -> 8.16
   - Update Surf dependency: 8.6 -> 8.8

### CODE REVIEW

**Review by araschitor — APPROVED** [review]

File: share/src/main/resources/alfresco/slingshot-application-context.xml [hunk 1]
```diff
@@ -157,6 +157,16 @@
             <null/>
         </property>
         <property name="proxyControllerInterceptor" ref="slingshotProxyControllerInterceptor"/>
+        <property name="uriWhitelist">
+            <list>
+                <value>^.*$</value>
+            </list>
+        </property>
+        <property name="uriBlacklist">
```
Comment (pieCit87) [inline share/src/main/resources/alfresco/slingshot-application-context.xml:165]: Is possible to skip this set up being already as default?

Comment (alexbalmus) [reply share/src/main/resources/alfresco/slingshot-application-context.xml:165]: I thought I'd leave it for informative reasons.

Comment (pieCit87) [reply share/src/main/resources/alfresco/slingshot-application-context.xml:165]: Good Point. I think here a doc can help.. @DavidCervigni @skopf guys what do you think? the point is to declare or not into the prop bean the blacklist with the default value.

**Review by pieCit87 — APPROVED** [review]
LGTM

**Pull request merged**